Tessian/resources/logs/tessian_logs.txt

```diff
@@ -1,186 +1,200 @@
-{"message_id":"df1e0e91-469d-4846-8875-39df98400455","module":"constructor","number_of_attachments":0,"threat_classification":null,"timestamp":"2021-07-13T09:39:12.21915Z","user_interaction":null,"user":"jane@example.com","alert_type":"warn","changes_made":"None: email not sent","check_performed_by":"Tessian Add-in","final_outcome":"Email not sent","subsequent_action":"Email reviewed by user and not sent","misdirected_email_prevented":null,"suggested_recipient":null,"filter_name":"Internal Communications","threat_types":null,"anomalous_recipient":null,"impersonation_type":null,"impersonated_address":null,"impersonated_domain":null,"email_is_sensitive":null,"unauthorized_email_prevented":null,"flag_reason":null,"intents":null,"email_summary":null,"unauthorised_recipients":null,"request_for_override":null,"attachments_total_size":0,"trigger_id":"141059-128", "updated_at": "2021-07-13T09:39:12.21915Z"}
+{"architect_details": {"breach_prevented": null, "final_outcome": "SENT_WITHOUT_CHANGES", "is_sensitive": false, "justifications": [], "triggered_logic_types": ["KEYWORD_PATTERNS", "REGEX"], "triggered_policy_ids": ["cc25c808-df24-4e21-ac99-2a347e11ae61"], "triggered_policy_names": ["Architect E2E Warn"], "user_responses": ["SEND"], "user_shown_message": true}, "created_at": "2022-09-29T00:06:53.836812Z", "id": "architect::outbound-3338690", "outbound_email_details": {"attachments": {"bytes": 0, "count": 0, "names": []}, "from": "andy@rotepad.com", "message_id": null, "recipients": {"all": ["architect_testing@rotepad.com"], "bcc": [], "cc": [], "count": 1, "to": ["architect_testing@rotepad.com"]}, "reply_to": [], "send_time": "2022-09-29T00:06:53.536451Z", "subject": "Test", "tessian_action": "SILENTLY_TRACK", "tessian_id": "dff6731c-57b7-444b-9eb2-92cbe1e0fff4", "transmitter": "andy@rotepad.com"}, "portal_link": "https://architest.tessian.xyz/0/events/architect?check_log_id=3338690&utm_source=external_api&utm_medium=trigger&utm_campaign=architect", "type": "architect", "updated_at": "2022-09-29T00:06:54.084847Z"}
```

Comment:
I tested the app with these logs but some of the dashboards panels are not loading up due to lack to sample data. Example is Overview/Suspicious Emails Detected

Reply:
Can you provide all logs which will light up all dashboards

Reply:
I can look at providing example logs for all dashboards on Monday.

Reply:
[2bd81df](https://github.com/SumoLogic/sumologic-public-partner-apps/pull/55/commits/2bd81df2749b7da330543dcea0b18e663e1ed63f)